modules/appeals_api/app/serializers/appeals_api/evidence_submission_serializer.rb

```diff
@@ -10,17 +10,18 @@ class EvidenceSubmissionSerializer < ActiveModel::Serializer
 
     attributes :id, :status, :code, :detail, :appeal_type, :appeal_id, :location, :created_at, :updated_at
 
+    delegate :status, to: :object
+    delegate :code, to: :object
+    delegate :detail, to: :object
```

Comment:
Delegating `detail` doesn't do anything if you later supply a local `#detail` method (your method will override the delegation). You can remove this delegate.